workflow/Snakefile

```diff
@@ -7,7 +7,12 @@ include: "rules/lst-data-selection.smk"
 import json
 from itertools import chain
 
-with open("configs/lst_agn.json", "r") as f:
+with open("build/runs.json", "r") as f:
+    runs = json.load(f)
+
+RUN_IDS = sorted(set(chain(*runs.values())))
```

Comment:
do you need that? It should be included via local.smk

Reply:
nope, still error from rebase